src/screens/HyperSwitch/Customers/Customers.res

```diff
@@ -0,0 +1,48 @@
+@react.component
+let make = () => {
+  open APIUtils
+  open CustomersEntity
+  let fetchDetails = useGetMethod()
+  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Loading)
+  let (customersData, setCustomersData) = React.useState(_ => [])
+  let (offset, setOffset) = React.useState(_ => 0)
+
+  let getCustomersList = async () => {
+    try {
+      setScreenState(_ => PageLoaderWrapper.Loading)
+      let customersUrl = getURL(~entityName=CUSTOMERS, ~methodType=Get, ())
```

Comment:
please check what is limit that we need to fetch

Reply:
Added - 10000